### PR TITLE
CCV-517: Group Messages

### PR DESCRIPTION
**Pull request by dwee9**

### CODE REVIEW

File: iOS/ViewControllers/ClassGroup/CourseGroupJoinController.cs [hunk 1]
```diff
@@ -152,16 +155,16 @@ private void joinCourse()
                     {
                         case CourseController.JoinCourseResponse.Success_SecurityIssue:
                         case CourseController.JoinCourseResponse.Success:
-                            ShowAlert( "Success", "You are now a member of this course.", ( ) => { reloadPersonData( ); } );
+                            ShowAlert( "Success", $"You are now a member of {details.name}.", ( ) => { reloadPersonData( ); } );
                             break;
                         case CourseController.JoinCourseResponse.AlreadyInCourse:
                             UIBlocking.Instance.stop( );
```
Comment (jhawkzz) [inline iOS/ViewControllers/ClassGroup/CourseGroupJoinController.cs]: Per @kkestell suggestion for security, can we make this message be the same as the normal "you joined" message?

File: iOS/ViewControllers/ClassGroup/CourseGroupJoinController.cs [hunk 2]
```diff
@@ -123,17 +123,20 @@ private void joinGroup()
...
                     {
-                        case GroupController.JoinGroupResponse.Success_SecurityIssue:
                         case GroupController.JoinGroupResponse.Success:
-                            ShowAlert("Success", "You are now a member of this group.", ( ) => { reloadPersonData( ); });
+                            ShowAlert($"You’ve successfully joined {details.name}", " Expect to hear from the coach in the next 24-48 hours!", ( ) => { reloadPersonData( ); });
                             break;
                         case GroupController.JoinGroupResponse.AlreadyInGroup:
                             UIBlocking.Instance.stop( );
-                            ShowAlert( "You are already a part of this group!", "" );
+                            ShowAlert( $"It looks like you’ve already joined {details.name}.", "Please contact the group coach if you need further assistance." );
```
Comment (jhawkzz) [inline iOS/ViewControllers/ClassGroup/CourseGroupJoinController.cs]: Per @kkestell suggestion for security, can we make this message be the same as the normal "you joined" message?

File: iOS/ViewControllers/ClassGroup/CourseGroupJoinController.cs [hunk 3]
```diff
@@ -152,16 +155,16 @@ private void joinCourse()
                     {
                         case CourseController.JoinCourseResponse.Success_SecurityIssue:
                         case CourseController.JoinCourseResponse.Success:
-                            ShowAlert( "Success", "You are now a member of this course.", ( ) => { reloadPersonData( ); } );
+                            ShowAlert( "Success", $"You are now a member of {details.name}.", ( ) => { reloadPersonData( ); } );
                             break;
                         case CourseController.JoinCourseResponse.AlreadyInCourse:
                             UIBlocking.Instance.stop( );
-                            ShowAlert( "You are already a part of this course!", "" );
+                            ShowAlert( $"It looks like you are already a member of {details.name}", "" );
```
Comment (jhawkzz) [inline iOS/ViewControllers/ClassGroup/CourseGroupJoinController.cs]: Per @kkestell suggestion for security, can we make this message be the same as the normal "you joined" message?

Comment (dwee9) [reply iOS/ViewControllers/ClassGroup/CourseGroupJoinController.cs]: Yup I was just reading over that conversation after I submitted this.  I will change those

Comment (jhawkzz) [reply iOS/ViewControllers/ClassGroup/CourseGroupJoinController.cs]: Thanks man, appreciate it!

**Pull request merged**